spec/rspec_puppet_facts_spec.rb

```diff
@@ -428,24 +428,25 @@
 
       it 'returns supported OS' do
         expect(subject.keys.sort).to eq [
-          'openbsd-5.7-amd64',
+          'openbsd-7.5-amd64',
```

Comment:
I wasn't 100% sure what's correct here. this passes now, but the filename is `openbsd-7-x86_64.facts`.